vizro-core/changelog.d/20240112_161730_huong_li_nguyen_split_up_demo.md

```diff
@@ -0,0 +1,45 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Highlights ✨
+
+- A bullet item for the Highlights ✨ category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+
+### Removed
+
+- Remove `dict` and `json` configurations from Gapminder `Demo` dashboard. ([#267](https://github.com/mckinsey/vizro/pull/267))
+
+### Added
+
+- Add `Features` compendium dashboard to `examples`. ([#267](https://github.com/mckinsey/vizro/pull/267))
+
```

Comment:
None of this is part of the released package so I don't think belongs in the changelog at all.